Silverchair.js

```diff
@@ -632,6 +644,156 @@ var testCases = [
 		"type": "web",
 		"url": "https://jov.arvojournals.org/issues.aspx?issueid=934904&journalid=178#issueid=934904",
 		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "https://academic.oup.com/book/26783/chapter/195715678",
+		"items": [
+			{
+				"itemType": "bookSection",
+				"title": "1Statistical inference with probabilistic graphical models",
```

Comment:
I wasn't sure here -- I could obviously try to remove the chapter numbers in the beginning, but potentially fraught -- titles can start with numbers (*"3-D imaging in modern medicine"